docs/user/migration/external-name-alias.md

```diff
@@ -0,0 +1,148 @@
+# Migration Guide for External Name Alias's Behavior Change
+
+## Context
+
+Istio with 1.21 version has changed the behavior of the Service of type ExternalName.
+Previously, for the Service of type ExternalName a separate `cluster` in the Envoy config was created.
```

Comment:
```suggestion
```

Reply:
Why do we want to remove that?

Reply:
Especially line bellow we have a continuation, so removing this one would make text broken

Reply:
I think it's important to mention that in the context

Reply:
This seems kind of out of place to me, as what the `cluster` thing does is never explained. And I have doubts that Istio module users understand what that means.

Reply:
I think for advanced user it might be beneficial to understand better the root of the change. For less advanced users, it does not hurt possibility to understand the rest, and can be easily skipped as a detail.